sig-testing/charter.md

```diff
@@ -0,0 +1,82 @@
+# SIG Testing Charter
+
+This charter adheres to the conventions described in the 
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+SIG Testing is interested in effective testing of Kubernetes.  We do not write or
+troubleshoot the project's tests, but instead focus on tooling that makes it
+easier for the community to write and run tests, and to contribute, analyze and
+act upon test results.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Project CI and merge automation via tools such as [prow] and [tide]
+- Extraction of test results from GCS and populating a public accessible
+  BigQuery dataset via [kettle]
+- Display and analysis of test artifacts via tools like [gubernator], 
+  [testgrid], [triage] and [velodrome]
+- Configuration management of jobs and ensuring they use a consistent
+  process via tools such as [job configs], [kubetest]
+- Tools that facilitate local testing of kubernetes such as [greenhouse] 
+  and [kind]
```

Comment:
+planter?

Reply:
ack, planter is what I meant, not greenhouse

Reply:
greenhouse, github cache etc are probably also in scope under another area ~nebulously infrastructure / automation, not sure how many things we really need to call out though.